features.json

```diff
@@ -187,7 +188,6 @@
 				"saturatedFloatToInt": true,
 				"signExtensions": true,
 				"simd": "2.0",
-				"threads": null,
```

Comment:
I don't know if it's accurate that they *don't* have threads support, but it's at least more accurate than saying that threads support is **not applicable**.

Reply:
Seems like this was semi intentional considering there was no way to spawn threads (https://github.com/WebAssembly/website/pull/313#issuecomment-1422303463), but this isn't the case anymore. There's a Rust target `wasm32-wasi-preview1-threads` that supports spawning threads and wasmtime supports it.

Reply:
Yeah, if there's now a way to spawn and use threads in these runtimes, I think it makes sense to mark this as supported. The exact mechanism for that (web workers, a WASI or even a nonstandard interface) is outside the scope of the core spec, and probably outside the scope of this page as it's currently designed. It might be helpful to find a way to link to some information (maybe just as an annotation alongside our long list of flags or something) but I think what you have proposed here is still an improvement over the current version. (and I think I'd accept a PR marking threads as supported behind a flag or whatever).